Content.Shared/_RMC14/Construction/RMCConstructionSystem.cs

```diff
@@ -144,6 +344,12 @@ public bool CanBuildAt(EntityCoordinates coordinates, string prototypeName, out
             return false;
         }
 
-        return true;
+        if (_rmcMap.HasAnchoredEntityEnumerator<BarricadeComponent>(coordinates, facing: direction.AsFlag()))
```

Comment:
This will crash when anchoring barricades, the default value needs to not be invalid or it needs to account for it, as flag throws an exception when it is